README.md

```diff
@@ -30,4 +52,37 @@ yield scrapy.Request("http://books.toscrape.com/",
                        "echoData": {"something": True}
                    }
                }),
+```
+
+## Example Code:
```

Comment:
Instead of adding this new section here, we can simply reuse the **How to use** section.

Reply:
Yea sure, I will add the example code on the **How to use**  section itself.

Reply:
Quick heads up! You might need to rebase your branch since we've converted the README from `md` to `rst` in https://github.com/scrapy-plugins/scrapy-zyte-api/pull/6. Since PyPI doesn't play well with `md` unfortunately.

Reply:
Yes, I have forked again and made the changes. Do I have to do open a new Pull Request again with the newly forked repo?

Reply:
You can simply add new commits on your existing PR.

Reply:
Sorry here you go https://github.com/scrapy-plugins/scrapy-zyte-api/pull/7

Since I forked and deleted my forked repo. I could not add new commits to this PR from my newly forked repo.